build-phases/README.md

```diff
@@ -1,23 +1,44 @@
-# Build phases
+# Gradle Kata: Build phases
```

Comment:
@michaelin why do we want to say "Gradle kata" when we are in a repo called just that?

Reply:
I'd like to align the format of the Gradle katas to that used for the Git katas.

Reply:
Fair choice :)